ansible/playbooks/create-meter.yaml

```diff
@@ -44,27 +44,44 @@
       shell: pwgen 10 1 > /energy/client_pass
       args:
         creates: /energy/client_pass
+    - name: clean old parity accounts
+      shell: rm -rf /energy/parity/datadir/keys/*
     - name: create new account
       shell: parity --chain /energy/parity/res/genesis.json --keys-path /energy/parity/datadir//keys account new --password /energy/client_pass > /energy/client_addr
     - name: get money to the account
       shell: curl -X POST -d `cat /energy/client_addr` http://146.169.47.73:6000/money
       ignore_errors: yes
```

Comment:
Run `sed -i '' '/\[account\]*$/,$d' res/config.toml` to truncate old account

Reply:
account -> config

Reply:
(y)

Reply:
This doesn't work becasue GNU sed != BSD sed

Reply:
Oops

Reply:
Resolved

Reply:
Do we need to fix `deploy-vm.sh` as well?

Reply:
Don't think so, we're not using it on VM's and on OSX it's fine

Reply:
@vincepandolfo uses Linux

Reply:
Eh, local fix? :D Anyway that doesn't belong to this PR